fieldservice/__manifest__.py

```diff
@@ -8,7 +8,7 @@
     "category": "Field Service",
     "author": "Open Source Integrators, Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/field-service",
-    "depends": ["base_territory", "base_geolocalize", "resource", "contacts"],
+    "depends": ["base_territory", "base_geolocalize", "resource", "contacts", "mail"],
```

Comment:
I don't believe we need to explicitly add the `mail` dependency.  It is already implied through the `contacts` dependency